modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -13,6 +13,19 @@ def cleanupGit() {
 }
 
 /**
+ * Checkout repo using SSH key
+ */
+def checkoutSSH(String repository, String org = 'alphagov', String url = 'github.com') {
+  checkout([$class: 'GitSCM',
+    branches: scm.branches,
+    userRemoteConfigs: [[
+      credentialsId: 'govuk-ci-ssh-key',
+      url: "git@${url}:${org}/${repository}.git"
```

Comment:
Because of `${org}/${repository}`, this will probably only work for GitHub instances, so maybe checkoutFromGitHubWithSSH would be a more descriptive name?